src/components/Favorites.tsx

```diff
@@ -1,9 +1,28 @@
-import MoviesList from './MoviesList';
+import { useEffect, useState } from 'react';
+import api from 'src/service/api';
+import RenderResults from './RenderResults';
 
 const Favorites = () => {
+  const [data, setData] = useState([{
+    poster_path: '',
+    original_title: '',
+    title: '',
+    vote_average: 0,
+    vote_count: '',
+    overview: '',
+    backdrop_path: '',
+    id: 0,
+  }]);
```

Comment:
why not empty array? why do we need to have these dummy values?

Reply:
discussed

Reply:
Still, add todo as discussed

Reply:
@mrSelieznov any updates on this PR?